### PR TITLE
Ratings workflow updated

### PR DESCRIPTION
**Pull request by Scttpr**
Full feature with new pages, stats on dashboard.vue and fixes according to PR comments

Checklist

- [x] I've added a mention in CHANGELOG.md

### CODE REVIEW

**Comment by Scttpr** [comment]
[Gérer les appréciations pour nous et par base](https://trello.com/c/ilbIqHUG/305-g%C3%A9rer-les-appr%C3%A9ciations-pour-nous-et-par-base)

File: components/charts/average.vue [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      switch (true) {
+        case (this.average < 25):
+          return 'low';
+        case (this.average >= 25 && this.average <= 75):
+          return 'medium';
+        case (this.average > 75):
+          return 'high';
+        default:
+          return '';
+      }
```
Comment (Nainterceptor) [inline components/charts/average.vue]: switch(true) seems to be fun but, it's strictly useless here.
```suggestion
      if (this.average < 25) {
        return 'low';
      } else if (this.average <= 75) {
        return 'medium';
      } else if (this.average > 75) {
        return 'high';
      }
      return '';
```

File: components/dashboard/ratings.vue [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+      <average-chart :average="average(ratings.recommendation) / 10 * 100" />
+    </template>
+  </fragmented-tile>
+</template>
+
+<script>
+import fragmentedTile from '~/components/dashboard/fragments.vue';
+import bgColorsMixin from '~/components/dashboard/mixins/background-colors';
+import pieChart from '~/components/charts/pie';
+import averageChart from '../charts/average.vue';
```
Comment (Nainterceptor) [inline components/dashboard/ratings.vue]: if you're using `~` to load pieChart, you can use the same pattern to load this one.

File: components/dashboard/ratings.vue [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+  components: {
+    fragmentedTile,
+    pieChart,
+    averageChart,
+  },
+  mixins: [bgColorsMixin],
+  props: {
+    ratings: {
+      type: Object,
+      default: () => null,
```
Comment (Nainterceptor) [inline components/dashboard/ratings.vue]: I'm pretty sure that you can do 
```suggestion
      default: null,
```

**Pull request merged**